### ISSUE TITLE
Can I get rid of sammy.js?

### ISSUE THREAD

**Issue by megabulk** [unveillance/CameraVFrontend#2]
I'm working on the main tabbed interface, and I want to be able to preserve state, so I need to make a URL like http://localhost:8888/#metadata
But this makes Sammy unhappy.
Are we actually using Sammy for anything? I think I'd decided that it could be entirely replaced by Backbone.

**Comment by harlo** [comment]
yes, I was pleased that we moved that functionality entirely to backbone.
please proceed!
